src/odemis/driver/test/orsay_test.py

```diff
@@ -2026,7 +2228,8 @@ def connector_test(test_case, va, parameters, valuepairs, readonly=False, hw_saf
             for a in attributes:
                 setattr(parameters[i], a, par_value1[i])
         sleep(settletime)
-        test_case.assertEqual(va.value, va_value1, "The second assertEqual between va.value and va_value1 isn't correct.")
+        test_case.assertEqual(va.value, va_value1,
+                              "The second assertEqual between va.value and va_value1 isn't correct.")
```

Comment:
I know you didn't change it this PR, but these messages don't seem to add any information to me.